tests/test_utils.py

```diff
@@ -481,3 +482,12 @@ def test_upgrade_pyproject_toml(tmpdir):
             "optional-dependencies": {"tests": ["pytest<=4.0.0,>=1.0.0"]},
         }
     }
+
+
+def test_isin_case_dashhyphen_ins():
+    vals = ["pandas", "python-dateutil"]
+
+    assert _isin_case_dashhyphen_ins("Pandas", vals)
```

Comment:
Looks great. Just one minor ask. Can we add some normal cases where its like `python-dateutil` to make sure `-` also come back the same

Reply:
Updated!